fe/model.py

```diff
@@ -60,11 +69,16 @@ def predict(self, ff_params: list, mol_a: Chem.Mol, mol_b: Chem.Mol, core: np.nd
         core: np.ndarray
             N x 2 list of ints corresponding to the atom mapping of the core.
 
+        callback: function
+            accepts a list of SimulationResults and a string, and doesn't return anything. may save to disk
+            TODO: is there a way to save intermediate results to disk that doesn't require passing a callable a few layers deep here...
```

Comment:
maybe document that link to the jax page that shows how to use jax registration of objects to actually do this properly so we can clean this up later easier?

Reply:
Resolved in https://github.com/proteneer/timemachine/pull/360/commits/9e72088cc1131049a840d0a4b58f596e05cb4532